etc/greenboot/greenboot.conf

```diff
@@ -12,4 +12,11 @@ GREENBOOT_WATCHDOG_CHECK_ENABLED=true
 ### This variable is the number of hours after an upgrade that we consider
 ### the new deployment as culprit of reboot.
 ### It has to be a positive integer. Defaults to 24 (hours).
-# GREENBOOT_WATCHDOG_GRACE_PERIOD=24
\ No newline at end of file
+# GREENBOOT_WATCHDOG_GRACE_PERIOD=24
+
+### This variable allows you to specify healthchecks to be skipped. 
+### The healthcheck must be specified by script name, as in the 
+### example below. Multiple healthchecks may be skipped by separating
+### the script names with spaces.
+### DISABLED_HEALTHCHECKS=("01_repository_dns_check.sh 02_watchdog.sh")
```

Comment:
Shouldn't this be `DISABLED_HEALTHCHECKS=('01_repository_dns_check.sh' '02_watchdog.sh')`?

Reply:
The way I have it currently written I believe it's correct, it takes in a string with multiple healthchecks and then uses spaces to delimit that string. If it's agreed that your method makes more sense then I can work to change it.

Reply:
You are using an array actually, because the string is between parentheses. So you need to use one of 
```bash
DISABLED_HEALTHCHECKS="01_repository_dns_check.sh 02_watchdog.sh"
```
or
```bash
DISABLED_HEALTHCHECKS=("01_repository_dns_check.sh"  "02_watchdog.sh")
``` 

I personally prefer the second approach because you don't execute any external commands, the array can be written in multiple lines and because using files with spaces works as expected:
```bash
DISABLED_HEALTH_CHECKS=(
  "one"
  "TWO"
  "THREE"
  "this is a script with spaces"
);

for DISABLED_HEALTH_CHECK in "${DISABLED_HEALTH_CHECKS[@]}"; do
    echo $DISABLED_HEALTH_CHECK ;
done
```

Reply:
Currently rewriting to use the second approach you mentioned here, I agree that is probably a simpler method. It will be included in the next force push

Reply:
Addressed in the most recent updates, resolving this issue